test/src/tests-identity.js

```diff
@@ -3084,6 +3086,46 @@ describe('identity', function() {
         done();
     });
 
+    describe('#identify', function() {
+        it("should set fst when the user does not change, after an identify request", function(done) {
+            mParticle._resetForTests(MPConfig);
+
+            const cookies = JSON.stringify({
+                gs: {
+                    sid: 'fst Test',
+                    les: new Date().getTime(),
+                },
+                current: {},
+                cu: 'current',
+            });
+    
+            mockServer.respondWith(urls.identify, [
+                200,
+                {},
+                JSON.stringify({ mpid: 'current', is_logged_in: false }),
+            ]);
+    
+            // We set the cookies because the init process will load cookies into
+            // our in-memory store, which we will check before and after the identify request
+            setCookie(workspaceCookieName, cookies, true);
+            mParticle.config.useCookieStorage = true;
+
+            mParticle.init(apiKey, mParticle.config);
+
+            expect(
+                mParticle.getInstance()._Store.getFirstSeenTime('current')
+            ).to.equal(null);
```

Comment:
Can you try the following?  Comment out `sid` and `les` from the cookies above ([line 3095-3096](https://github.com/mParticle/mparticle-web-sdk/pull/876/files#diff-f9b04257df2922fb4c4451e6ba6453eb1ab9dd5c51f3e148939dca7986bed66aR3115-R3117) in this commit)

Does the test in 3115 fail?  

I think it would. Because normally `init` will call `identify`, and the reason it is not calling it here, and you have to manually call it below this line, is because there is an `sid`, and/or `les`.  If you step through, I think it will skip the `identify` call because of that.

If the above is in fact the case, we should add a comment above `mParticle.init` to explain why it isn't called, to help future developers out.